docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -1,9 +1,38 @@
 
+tag::allow-no-indices[]
+`allow_no_indices`::
+(Optional, boolean) If `true`, the request returns an error if a wildcard
+expression or `_all` value retrieves only missing or closed indices. This
+parameter also applies to <<indices-aliases,index aliases>> that point to a
+missing or closed index.
+end::allow-no-indices[]
```

Comment:
@jrodewig It seems this behaves opposite of the description here. afaict `allow_no_indices = true` will allow empty indices and thus not return an error. `allow_no_indices = false` will not allow empty and will return an error if empty indicies are encountered. 

cc @weltenwort

Reply:
Thanks for catching this @sqren. I've opened #46450 to correct.